machine-check/examples/example.test.ts

```diff
@@ -1,5 +1,5 @@
 import { describe, expect, it } from '@jest/globals'
-import { Door, Control, HangarBay } from './example-proto.js'
+import { Door, Control } from './example-proto'
```

Comment:
I thought that ESM requires the `.js` suffix?

Reply:
You're right. I was tinkering with this and forgot to revert. I'm curious though how `tsc` perceive this compared to `.js`